src/blockscoutApi/index.ts

```diff
@@ -169,4 +172,31 @@ export class BlockscoutAPI extends DataSource {
       .then(({ data }) => data.result)
       .catch(() => [])
   }
+
+  async getNftHoldersData ({ address, nextPageParams }: GetNftHoldersData) {
+    const url = `${this.url}/v2/tokens/${address.toLowerCase()}/instances`
+    console.log('getting nft holders data')
+    try {
+      const response = await this.axios?.get<ServerResponseV2<NftTokenHoldersResponse>>(url, {
+        params: nextPageParams
+      })
+
+      if (response?.status === 200) {
+        const transformedResponse = nftHolderTransformedData(response.data.items)
```

Comment:
Maybe we could rename this; another option could be mapToNftHolder
```suggestion
        const nftHolders = transformResponseToNftHolder(response.data.items)
```